src/Codebreaker.WPF/Views/Components/GameResultDisplay.xaml.cs

```diff
@@ -0,0 +1,74 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+
+namespace Codebreaker.WPF.Views.Components;
+
+/// <summary>
+/// Interaction logic for GameResultDisplay.xaml
+/// </summary>
+[ObservableObject]
+public partial class GameResultDisplay : UserControl
+{
+    public GameResultDisplay()
+    {
+        InitializeComponent();
+    }
+
+    public GameMode GameMode
+    {
+        get => (GameMode)GetValue(GameModeProperty);
+        set
+        {
+            SetValue(GameModeProperty, value);
+            Debug.WriteLine(value);
+            OnPropertyChanged(nameof(GameMode));
+        }
+    }
+
+    public static readonly DependencyProperty GameModeProperty =
+        DependencyProperty.Register(nameof(GameMode), typeof(GameMode), typeof(GameResultDisplay), new PropertyMetadata(GameMode.NotRunning, OnGameModeChanged));
+
+    public ICommand? LostCommand
```

Comment:
are these commands bound from within the user control or outside?
if it's just from within the user control, and this code is just related with the UI and not outside functionality is needed, click events would simplify the code here.